package.json

```diff
@@ -22,8 +30,8 @@
   },
   "homepage": "https://github.com/NYTimes/kyt#readme",
   "dependencies": {
-    "jest": "16.0.2",
     "ps-tree": "1.1.0",
-    "shelljs": "0.7.5"
+    "shelljs": "0.7.5",
+    "jest": "17.0.3"
```

Comment:
should we upgrade Jest to 17 in kyt-core too? There may be some breaking changes we'll want to look into. How much do we care about keeping them in sync?